### PR TITLE
Localise more text

### PR DESCRIPTION
**Pull request by Jawshy**
#### Changes proposed in this pull request:
- Localise text. 
  - See individual commit comments and diff for more details. 
- Added whitespace. 

---
#### Todo:
- [ ] **Fully translate** 
  - Help by posting the correct translation as a line comment. 
- [ ] **Pass test** 
  - Help by testing for errors. 
- [ ] **Fully localise** 
  - Help by finding text yet to be localised.

### CODE REVIEW

**Comment by BoGuu** [comment]
0/3 ;(

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -11175,6 +11175,17 @@
             <Russian>Вы были обнаружены для мошенничества и было сообщено на сервер. Удачного дня.</Russian>
             <German>Sie wurden für Betrug erkannt und haben an den Server gemeldet. Genieße deinen Tag.</German>
```
Comment (DgH5001) [inline Altis_Life.Altis/stringtable.xml]: Sie wurden als Hacker erkannt und sind an den Server gemeldet worden. Genießen sie ihren Tag.

File: Altis_Life.Altis/stringtable.xml [hunk 2]
```diff
@@ -11197,6 +11197,17 @@
             <Russian><t align='center'><t color='#FF0000'><t size='3'>SPY-GLASS</t></t><br/>Cheater Flagged</t><br/><br/>Name: %1<br/>Detection: %2</Russian>
             <German><t align='center'><t color='#FF0000'><t size='3'>SPY-GLASS</t></t><br/>Cheater Flagged</t><br/><br/>Name: %1<br/>Detection: %2</German>
```
Comment (DgH5001) [inline Altis_Life.Altis/stringtable.xml]: <German><t align='center'><t color='#FF0000'><t size='3'>SPY-GLASS</t></t><br/>Hacker erkannt</t><br/><br/>Name: %1<br/>Erkenntniss: %2</German>

File: Altis_Life.Altis/stringtable.xml (hunk 1 above)
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: @setoy, is the [above translation](https://github.com/AsYetUntitled/Framework/pull/120#discussion_r87704972) more accurate than the current one?

Comment (setoy) [inline Altis_Life.Altis/stringtable.xml]: Better use:
`Du wurdest als Betrüger erkannt und an den Server gemeldet. Genieße deinen Tag.`

**Comment by BoGuu** [comment]
#177 #185 #179

**Comment by WolfSkin1337** [comment]
#185 is deleted
#195 is the new version

File: Altis_Life.Altis/stringtable.xml [hunk 3]
```diff
@@ -4353,6 +4353,17 @@
...
+        <Key ID="STR_NOTF_stoppedSpectating">
+            <Original>You have stopped spectating.</Original>
+            <Czech>Přestali jste divácké úlohy.</Czech>
+            <French>Vous avez arrêté spectating.</French>
+            <Spanish>Usted ha dejado de ser espectador.</Spanish>
+            <Italian>Hai smesso di spettatori.</Italian>
+            <Polish>Zatrzymaniu widzem.</Polish>
+            <Portuguese>Você parou espectador.</Portuguese>
+            <Russian>Вы остановили spectating.</Russian>
+            <German>Sie haben das Zuschauen gestoppt.</German>
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/stringtable.xml]: Du hast das Beobachten beendet.

File: Altis_Life.Altis/stringtable.xml [hunk 4]
```diff
@@ -4364,6 +4364,17 @@
...
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/stringtable.xml]: <German>Du hast das Beobachten beendet.</German>

File: Altis_Life.Altis/stringtable.xml [hunk 5]
```diff
@@ -4397,6 +4408,17 @@
...
+        <Key ID="STR_NOTF_emptyArray_levelCheck">
+            <Original>An empty array was passed to fn_levelCheck.sqf</Original>
+            <Czech>An empty array was passed to fn_levelCheck.sqf</Czech>
+            <French>An empty array was passed to fn_levelCheck.sqf</French>
+            <Spanish>An empty array was passed to fn_levelCheck.sqf</Spanish>
+            <Italian>An empty array was passed to fn_levelCheck.sqf</Italian>
+            <Polish>An empty array was passed to fn_levelCheck.sqf</Polish>
+            <Portuguese>An empty array was passed to fn_levelCheck.sqf</Portuguese>
+            <Russian>An empty array was passed to fn_levelCheck.sqf</Russian>
+            <German>An empty array was passed to fn_levelCheck.sqf</German>
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/stringtable.xml]: `<German>Ein leeres Feld wurde an fn_levelCheck.sqf übergeben.</German>`

File: Altis_Life.Altis/stringtable.xml [hunk 6]
```diff
@@ -11175,6 +11197,83 @@
...
+        <Key ID="STR_SpyDetect_cookieJar">
+            <Original>SPYGLASS-FLAG:%1:%2:%3</Original>
+            <Czech>SPYGLASS-FLAG:%1:%2:%3</Czech>
+            <French>SPYGLASS-FLAG:%1:%2:%3</French>
+            <Spanish>SPYGLASS-FLAG:%1:%2:%3</Spanish>
+            <Italian>SPYGLASS-FLAG:%1:%2:%3</Italian>
+            <Polish>SPYGLASS-FLAG:%1:%2:%3</Polish>
+            <Portuguese>SPYGLASS-FLAG:%1:%2:%3</Portuguese>
+            <Russian>SPYGLASS-FLAG:%1:%2:%3</Russian>
+            <German>SPYGLASS-FLAG:%1:%2:%3</German>
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/stringtable.xml]: `<German>SPYGLASS-ERKENNUNG:%1:%2:%3</German>`

File: Altis_Life.Altis/stringtable.xml [hunk 7]
```diff
@@ -11175,6 +11197,83 @@
...
+            <French><t align='center'><t color='#FF0000'><t size='3'>SPY-GLASS</t></t><br/>Cheater Flagged</t><br/><br/>Name: %1<br/>Detection: %2</French>
+            <Spanish><t align='center'><t color='#FF0000'><t size='3'>SPY-GLASS</t></t><br/>Cheater Flagged</t><br/><br/>Name: %1<br/>Detection: %2</Spanish>
+            <Italian><t align='center'><t color='#FF0000'><t size='3'>SPY-GLASS</t></t><br/>Cheater Flagged</t><br/><br/>Name: %1<br/>Detection: %2</Italian>
+            <Polish><t align='center'><t color='#FF0000'><t size='3'>SPY-GLASS</t></t><br/>Cheater Flagged</t><br/><br/>Name: %1<br/>Detection: %2</Polish>
+            <Portuguese><t align='center'><t color='#FF0000'><t size='3'>SPY-GLASS</t></t><br/>Cheater Flagged</t><br/><br/>Name: %1<br/>Detection: %2</Portuguese>
+            <Russian><t align='center'><t color='#FF0000'><t size='3'>SPY-GLASS</t></t><br/>Cheater Flagged</t><br/><br/>Name: %1<br/>Detection: %2</Russian>
+            <German><t align='center'><t color='#FF0000'><t size='3'>SPY-GLASS</t></t><br/>Cheater Flagged</t><br/><br/>Name: %1<br/>Detection: %2</German>
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/stringtable.xml]: `<German><t align='center'><t color='#FF0000'><t size='3'>SPY-GLASS</t></t><br/>Betrüger Erkannt</t><br/><br/>Name: %1<br/>Erkennung: %2</German>`

File: Altis_Life.Altis/stringtable.xml [hunk 8]
```diff
@@ -11175,6 +11197,83 @@
...
+            <Polish>%1 was observed by SPY-GLASS, he/she was trying to access commanding menu:\n\n %2\n\n and that commanding-menu is not known to the system. PLEASE NOTE he/she may not be cheating but the SPY-GLASS found it relevant to report in.</Polish>
+            <Portuguese>%1 was observed by SPY-GLASS, he/she was trying to access commanding menu:\n\n %2\n\n and that commanding-menu is not known to the system. PLEASE NOTE he/she may not be cheating but the SPY-GLASS found it relevant to report in.</Portuguese>
+            <Russian>%1 was observed by SPY-GLASS, he/she was trying to access commanding menu:\n\n %2\n\n and that commanding-menu is not known to the system. PLEASE NOTE he/she may not be cheating but the SPY-GLASS found it relevant to report in.</Russian>
+            <German>%1 was observed by SPY-GLASS, he/she was trying to access commanding menu:\n\n %2\n\n and that commanding-menu is not known to the system. PLEASE NOTE he/she may not be cheating but the SPY-GLASS found it relevant to report in.</German>
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/stringtable.xml]: `<German>%1 wurde beobachtet durch SPY-GLASS. Er/Sie hatte versucht auf das Befehlsmenü:\n\n %2\n\n zuzugreifen, diese ist dem System aber nicht bekannt. BITTE BEACHTE! Er/Sie betrügt deshalb nicht, aber SPY-GLASS fand es trotzdem relevant zu melden.</German>`

File: Altis_Life.Altis/stringtable.xml [hunk 9]
```diff
@@ -11175,6 +11197,83 @@
...
+        <Key ID="STR_SpyDetect_Variable_MN">
+            <Original>Variable: %1 is not allowed TYPE: %2 NS: MN</Original>
+            <Czech>Variable: %1 is not allowed TYPE: %2 NS: MN</Czech>
+            <French>Variable: %1 is not allowed TYPE: %2 NS: MN</French>
+            <Spanish>Variable: %1 is not allowed TYPE: %2 NS: MN</Spanish>
+            <Italian>Variable: %1 is not allowed TYPE: %2 NS: MN</Italian>
+            <Polish>Variable: %1 is not allowed TYPE: %2 NS: MN</Polish>
+            <Portuguese>Variable: %1 is not allowed TYPE: %2 NS: MN</Portuguese>
+            <Russian>Variable: %1 is not allowed TYPE: %2 NS: MN</Russian>
+            <German>Variable: %1 is not allowed TYPE: %2 NS: MN</German>
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/stringtable.xml]: `<German>Variable: %1 ist nicht erlaubt. TYPE: %2 NS: MN</German>`

File: Altis_Life.Altis/stringtable.xml [hunk 10]
```diff
@@ -11175,6 +11197,83 @@
...
+        <Key ID="STR_SpyDetect_Variable_UI">
+            <Original>Variable: %1 is not allowed TYPE: %2 NS: UI</Original>
+            <Czech>Variable: %1 is not allowed TYPE: %2 NS: UI</Czech>
+            <French>Variable: %1 is not allowed TYPE: %2 NS: UI</French>
+            <Spanish>Variable: %1 is not allowed TYPE: %2 NS: UI</Spanish>
+            <Italian>Variable: %1 is not allowed TYPE: %2 NS: UI</Italian>
+            <Polish>Variable: %1 is not allowed TYPE: %2 NS: UI</Polish>
+            <Portuguese>Variable: %1 is not allowed TYPE: %2 NS: UI</Portuguese>
+            <Russian>Variable: %1 is not allowed TYPE: %2 NS: UI</Russian>
+            <German>Variable: %1 is not allowed TYPE: %2 NS: UI</German>
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/stringtable.xml]: `<German>Variable: %1 ist nicht erlaubt. TYPE: %2 NS: UI</German>`

File: Altis_Life.Altis/stringtable.xml [hunk 11]
```diff
@@ -11175,6 +11197,83 @@
...
+        <Key ID="STR_SpyDetect_Observes">
+            <Original>||SPY-GLASS Observes|| Name: %1 | UID: %2 | Reason: %3</Original>
+            <Czech>||SPY-GLASS Observes|| Name: %1 | UID: %2 | Reason: %3</Czech>
+            <French>||SPY-GLASS Observes|| Name: %1 | UID: %2 | Reason: %3</French>
+            <Spanish>||SPY-GLASS Observes|| Name: %1 | UID: %2 | Reason: %3</Spanish>
+            <Italian>||SPY-GLASS Observes|| Name: %1 | UID: %2 | Reason: %3</Italian>
+            <Polish>||SPY-GLASS Observes|| Name: %1 | UID: %2 | Reason: %3</Polish>
+            <Portuguese>||SPY-GLASS Observes|| Name: %1 | UID: %2 | Reason: %3</Portuguese>
+            <Russian>||SPY-GLASS Observes|| Name: %1 | UID: %2 | Reason: %3</Russian>
+            <German>||SPY-GLASS Observes|| Name: %1 | UID: %2 | Reason: %3</German>
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/stringtable.xml]: `<German>||SPY-GLASS Beobachtung|| Name: %1 | UID: %2 | Grund: %3</German>`

File: Altis_Life.Altis/stringtable.xml [hunk 12]
```diff
@@ -11175,6 +11197,83 @@
...
+        <Key ID="STR_SpyDetect_VariableSetBeforeClientInitialized">
+            <Original>Variable set before client initialized: %1</Original>
+            <Czech>Variable set before client initialized: %1</Czech>
+            <French>Variable set before client initialized: %1</French>
+            <Spanish>Variable set before client initialized: %1</Spanish>
+            <Italian>Variable set before client initialized: %1</Italian>
+            <Polish>Variable set before client initialized: %1</Polish>
+            <Portuguese>Variable set before client initialized: %1</Portuguese>
+            <Russian>Variable set before client initialized: %1</Russian>
+            <German>Variable set before client initialized: %1</German>
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/stringtable.xml]: <German>Variable gesetzt bevor Client initialisiert: %1</German>

**Comment by Jawshy** [comment]
Thanks, @WolfSkin1337. Done in https://github.com/AsYetUntitled/Framework/pull/120/commits/0ef367a46b176fbd3b7545d87896cda69306874d.

**Comment by WolfSkin1337** [comment]
see from https://github.com/AsYetUntitled/Framework/blob/localise/life_server/functions.sqf#L243
to https://github.com/AsYetUntitled/Framework/blob/localise/life_server/functions.sqf#L308

there are text for translate also the `case 0-5`

**Comment by WolfSkin1337** [comment]
i have found another one:
`No Licenses`  -> `"STR_Cop_NoLicenses"`
https://github.com/AsYetUntitled/Framework/blob/localise/Altis_Life.Altis/core/pmenu/fn_p_updateMenu.sqf#L63

**Comment by WolfSkin1337** [comment]
and found one more
`[""realname"",""License Shop""];` at the [SQM´s](https://github.com/AsYetUntitled/Framework/tree/master/SQMs)

**Comment by Jawshy** [comment]
*Writes down on to-do list.*

**Comment by WolfSkin1337** [comment]
- [ ] /life_server/functions.sqf ([link](https://github.com/AsYetUntitled/Framework/blob/localise/life_server/functions.sqf#L243-L308))
- [ ] /Altis_Life.Altis/core/pmenu/fn_p_updateMenu.sqf ([link](https://github.com/AsYetUntitled/Framework/blob/localise/Altis_Life.Altis/core/pmenu/fn_p_updateMenu.sqf#L63))
- [ ] /SQMs/missionAltis.sqm (search `[""realname"",""License Shop""]`)

**Comment by NeFShARk** [comment]
@WolfSkin1337 

The "realname" variables in the SQMs, are the text that appears above the NPC when the player get in a certain distance from it, there are more! Not only in the "Licence Shop" NPC's. 

Like:

[""realname"",""Cop Vehicle Store""]; 
[""realname"", ""Car Shop""];

These variables(realname) are also used in the objects in the gang hideouts:

[""realname"",""Gang Armament""];
[""realname"",""Marijuana Processing""];

/\ But different from the NPC's, it doesn't work, nothing is displayed above the object in the game! This appear to only work on NPC's and not on objects. Maybe they should be removed? I mean the ones that are used in the objects? Or i am wrong?

File: Altis_Life.Altis/core/clientValidator.sqf [hunk 13]
```diff
@@ -10,16 +10,16 @@
...
+    "life_revive_fee", "life_gangPrice", "life_gangUpgradeBase", "life_enableFatigue", "life_paycheck_period", "life_vShop_rentalOnly", "sell_array", "buy_array",
+    "life_weapon_shop_array", "life_garage_prices", "life_garage_sell", "life_houseLimit", "life_gangUpgradeMultipler", "life_impound_car", "life_impound_boat",
     "life_impound_air"
 ];
 
 {
     if (!isNil {(missionNamespace getVariable _x)}) exitWith {
-        [profileName,getPlayerUID player,format ["VariableSetBeforeInitialized_%1",_x]] remoteExecCall ["SPY_fnc_cookieJar",RSERV];
-        [profileName,format ["Variable set before client initialized: %1",_x]] remoteExecCall ["SPY_fnc_notifyAdmins",RCLIENT];
+        [profileName,getPlayerUID player, format ["VariableSetBeforeInitialized_%1", _x]] remoteExecCall ["SPY_fnc_cookieJar", RSERV];
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/core/clientValidator.sqf]: `[profileName,getPlayerUID player, format ["VariableSetBeforeInitialized_%1", _x]] remoteExecCall ["SPY_fnc_cookieJar", RSERV];`
forget a spaces?
`[profileName, getPlayerUID player, format ["VariableSetBeforeInitialized_%1", _x]] remoteExecCall ["SPY_fnc_cookieJar", RSERV];`

**Pull request merged**